### DISCUSSION TITLE
Backup of user database

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#10160] [Unsupported Versions]
Is it possible to make a backup of the user database, then import or replace it on a new installation

**Comment by InfosecGoon** [comment]
The user database for the web interface is stored in /nsm/kratos/db/db.sqlite -- you should be able to move that to a new installation (assuming it's running the same version of SO) and have the same authentication information there.

**Reply by OzzyKampha** [reply]
Nice! I will try that out. I dont need the same secrets or anything else?
